### PR TITLE
Upgrade to Gatsby v3

### PR DESCRIPTION
**Pull request by jgarrow**
This PR upgrades the themes to Gatsby v3, upgrading other dependencies where necessary to account for breaking changes from the version bump.

## TO DO:
- [x] Fix enlarged header logo

### CODE REVIEW

File: packages/gatsby-theme-apollo-core/gatsby-config.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ module.exports = {
     {
       resolve: 'gatsby-plugin-less',
       options: {
-        modifyVars: mapKeys(colors, (value, key) => `color-${key}`)
+        lessOptions: {
+          modifyVars: mapKeys(colors, (value, key) => `color-${key}`)
+        }
```
Comment (jgarrow) [inline packages/gatsby-theme-apollo-core/gatsby-config.js:14]: The update for this plugin moves options to more specific [`lessOptions` and `loaderOptions`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-less#how-to-use)

File: packages/gatsby-theme-apollo-core/package.json [hunk 2]
```diff
@@ -13,12 +13,13 @@
...
     "@apollo/space-kit": "^5.6.0",
     "@emotion/core": "^10.0.7",
-    "@emotion/styled": "^10.0.7",
-    "@svgr/webpack": "^4.2.0",
-    "gatsby-plugin-emotion": "^4.0.3",
-    "gatsby-plugin-less": "^3.0.1",
-    "gatsby-plugin-react-helmet": "^3.0.2",
-    "gatsby-plugin-svgr": "^2.0.1",
+    "@emotion/react": "^11.0.0",
+    "@emotion/styled": "^11.0.0",
```
Comment (jgarrow) [inline packages/gatsby-theme-apollo-core/package.json:17]: The update for Emotion 11 and accompanying Gatsby plugin also require `@emotion/styled`. Not sure if we still need `@emotion/core` or if it's safe to remove.

File: packages/gatsby-theme-apollo-core/package.json [hunk 3]
```diff
@@ -13,12 +13,13 @@
...
+    "@svgr/webpack": "^5.0.0",
+    "gatsby-plugin-emotion": "^6.0.0",
+    "gatsby-plugin-less": "^5.0.0",
+    "gatsby-plugin-react-helmet": "^4.0.0",
+    "gatsby-plugin-svgr": "3.0.0-beta.0",
```
Comment (jgarrow) [inline packages/gatsby-theme-apollo-core/package.json:22]: There's an open PR to update this plugin to support Gatsby v3. Someone in the comments [published a beta version](https://github.com/zabute/gatsby-plugin-svgr/pull/51#issuecomment-789350978) in the interim until it gets merged.

File: packages/gatsby-theme-apollo-core/package.json (hunk 2 above)
Comment (jgarrow) [reply packages/gatsby-theme-apollo-core/package.json:17]: Update: the version of space-kit that we're using stil lrelies on @emotion/core, so I'm leaving it in

**Comment by trevorblades** [comment]
LGTM @jgarrow! 🥞

**Pull request merged**